contributing.md

```diff
@@ -138,17 +138,17 @@ We will support Python 3.12 and higher versions soon.
 
 ### Install requirements - (Non-Windows)
 ```bash
-(plotly_dev) $ pip install -r packages/python/plotly/requirements.txt
-(plotly_dev) $ pip install -r packages/python/plotly/optional-requirements.txt
+(plotly_dev) $ pip install -r packages/python/plotly/requires-install.txt
```

Comment:
Is this step required in contributing.md? I think it's covered by the editable install of `plotly` further down.

Reply:
@ndrezn @emilykl if you can spare time today, it would be great to get this one in - thanks

Reply:
good catch @LiamConnors , I'll remove this

(I verified that the setup instructions still work without this step, at least on Mac)